CHANGES.md

```diff
@@ -85,6 +85,7 @@
 
 * Fixed exception chaining issue in GCS connector (Python) ([#26769](https://github.com/apache/beam/issues/26769#issuecomment-1700422615)).
 * Fixed streaming inserts exception handling, GoogleAPICallErrors are now retried according to retry strategy and routed to failed rows where appropriate rather than causing a pipeline error (Python) ([#21080](https://github.com/apache/beam/issues/21080)).
+* Fixed a bug in Python SDK's cross-language Bigtable sink that mishandled records that don't have an explicit timestamp set: [#28632](https://github.com/apache/beam/issues/28632).
```

Comment:
We still need to add the known issues to 2.49.0 and 2.50. in this md file. I know it is a bit redundant. but it helps keeping all these files are sycned.

Reply:
Yup good call, best to have it clear everywhere.
Done